src/index.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+const fs = require('fs/promises');
+const { sep, basename } = require('path');
+
+const fileParam = process.argv[2];
+const endpointParam = process.argv[3];
+
+moveFile(fileParam, endpointParam);
+
+async function moveFile(file, endpoint) {
+  try {
+    const dataOfFile = await fs.readFile(file);
```

Comment:
The readFile and writeFile methods are being used to move a file, which can be inefficient for large files. Consider using the rename method instead.